test-util.sh

```diff
@@ -41,7 +41,7 @@ function check_pipeline() {
 
   echo "    check_gold $gold matches $@"
   local tmp="/tmp/icdiff.output"
-  bash -c "$*" &> "$tmp"
+  bash -c "$*" > "$tmp"
```

Comment:
SIZer prints debug messages to stderr, including PIDs of child processes, which may not be identical run-to-run. Seems like we probably don't want to require stderr consistency in tests, exactly because messages written to stderr may depend on system conditions?